contracts/core/extensions/CoreFactory.sol

```diff
@@ -80,7 +80,7 @@ contract CoreFactory is
         string _name,
         string _symbol
     )
-        public
+        external
```

Comment:
do we want to move this under another header? Right now its under `/* ============ Public Functions ============ */`